build.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Build script for Railway deployment
+echo "Building arda-poc application..."
+
+# Ensure we're in the right directory
+cd /app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)